sway.5.txt

```diff
@@ -175,3 +179,34 @@ Commands
 **workspace** <name> output <output>::
 	Specifies that the workspace named _name_ should appear on the specified
 	_output_.
+
+Criteria
+--------
+
+A criteria is a string in the form of e.g.:
+
+	[class="[Rr]egex.*" title="some title"]
+
+The string contains one or more (space separated) attribute/value pairs and they
+are used by some commands filter which views to execute actions on. All attributes
+must match for the criteria string to match.
+
+Currently supported attributes:
+
+**class**::
+	Compare value against the window class. Can be a regular expression. If value
+	is _focused_ then the window class must be the same as that of the currently
+	focused window.
+
+**id**::
+	Compare against the app id. Must be a normal string.
```

Comment:
Matches on the window id have two use cases:

1. i3 internally uses these when serializing state before an in-place restart, so that the same window will be matched when loading the state after the restart.

2. IPC users can use this criterion in commands after locating a specific window in the tree.